localization/english/event_localization/wc_liberate_lordaeron_events_l_english.yml

```
@@ -0,0 +1,11 @@
+﻿l_english:
+ 
+ liberate_lordaeron.1000.title:1 "Lordaeron Reborn"
+ liberate_lordaeron.1000.desc:0 "After countless deaths and conflicts, [GetTitleByKey('b_lordaeron').GetName] has been wrestled away from the Scourge's grasp.\n\nThe people of Lordaeron will settle back into their homelands and the plague shall be cleansed off... but now the choices are laid bare before us...\n\nWho shall inherit the new [GetTitleByKey('k_lordaeron').GetName]? I have taken back the Kingdom from the Scourge after all, why shouldn't I keep it to myself? Should I hand it back to the Menethils, to keep it stable? Or do I hand it to our Theocratic Leaders?"
```

```suggestion
 liberate_lordaeron.1000.desc:0 "After countless deaths and conflicts, the [GetTitleByKey('b_lordaeron').GetName] has been wrested from the Scourge's grasp. The people of these ravaged lands will settle into their homelands once again and the plague shall be cleansed from the air and soil, but now the problem of rulership is laid bare before us... Who shall inherit the new [GetTitleByKey('b_lordaeron').GetBaseNameNoTierNoTooltip]?\n\nPerhaps the right thing to do would be to hand it back to the Menethils, to keep it stable? Or perhaps it would be most pious of me to hand it to the highest leadership of our Holy Faith?\n\nThen again, it was I alone who took back the [GetTitleByKey('b_lordaeron').GetTierAsNameNoTooltip] from the Scourge. After all, why shouldn't I keep it for myself?"
```
did a little gymnastics to preserve the LOTR reference